.github/workflows/build.yml

```diff
@@ -0,0 +1,86 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main, develop ]
+
+jobs:
+  build:
```

Comment:
This job doesn't do anything.
It only changes make `./gradlew` executable (It should already), but it is not used at all.
The rest of the jobs will start their own container and this change is not used